testing/scenario_app/ios/FlutterAppExtensionTestHost/FlutterAppExtensionTestHost/AppDelegate.h

```diff
@@ -0,0 +1,9 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
Shouldn't these new files be using the standard engine boilerplate (`Copyright 2013 The Flutter Authors.`)?